client.py

```diff
@@ -364,8 +365,9 @@ def run_command(command):
                 sys.exit(0)
             elif strData[:3] == "msg":
                 MessageBox(strData[3:])
-            elif strData[:4] == "site":
```

Comment:
i got rid of the site option in the last few commits because it was pretty much useless.

Reply:
I thought the same thing but I didn't want to get rid of any features you put in, so I just improved it a little. I'll just get rid of it.